code.js

```diff
@@ -1,3 +1,33 @@
-function allPairsShortestPaths(graph) {
-    return [];
+
+function allPairsShortestPaths(graph) 
+{
+    var dist = Array(graph.length).fill().map(() => Array(graph.length).fill(Infinity))
+    
+    for (let u = 0; u < dist.length; u++)
+    {
+        if (dist[u].length == 1) dist[u][u] = Infinity
```

Comment:
Would probably make more sense to take whatever is in the adjacency matrix as the distance.